wordpress-mobile/WordPress-iOS
# Update WordPressKit to v4.6.0-beta.1 and Authenticator to 1.11.0-beta.1

### Changes
- update the Authenticator pod version
- update the WordPressKit pod version

Ref. https://github.com/wordpress-mobile/wpxmlrpc/pull/48

These pod updates are expected to be non-breaking changes. Both Authenticator and WordPressKit are getting updated so that Authenticator can use the latest version of `wpxmlrpc`. The latest version of `wpxmlrpc` has the recommended project setting changes so that Xcode warnings disappear.

Ref. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/196 and https://github.com/wordpress-mobile/WordPressKit-iOS/pull/225

**To test:**
1. Build and run the app. 
2. Log out.
3. Log in by adding a site address (any site that relies on XML-RPC / is not supported by Jetpack).
4. Remove site.

Logging in and out and verifying XML-RPC calls should act normally.

PR submission checklist:
